src/mono/wasm/debugger/BrowserDebugProxy/EvaluateExpression.cs

```diff
@@ -390,17 +390,21 @@ internal static async Task<JObject> CompileAndRunTheExpression(string expression
             if (expressionTree == null)
                 throw new Exception($"BUG: Unable to evaluate {expression}, could not get expression from the syntax tree");
 
-            try {
+            try
+            {
                 var newScript = script.ContinueWith(
                     string.Join("\n", findVarNMethodCall.variableDefinitions) + "\nreturn " + syntaxTree.ToString());
 
                 var state = await newScript.RunAsync(cancellationToken: token);
-
                 return JObject.FromObject(ConvertCSharpToJSType(state.ReturnValue, state.ReturnValue.GetType()));
             }
-            catch (Exception)
+            catch (CompilationErrorException cee)
             {
-                throw new ReturnAsErrorException($"Cannot evaluate '{expression}'.", "CompilationError");
+                throw new ReturnAsErrorException($"Running expression: {expression} failed with error: {cee.Message}. Check if the expression is correct.", "CompilationError");
```

Comment:
I think we can use the simpler message here - `Cannot evaluate '{expression}': {cee.Message}`